cypress/integration/dib-travel/personal-settings/profile/profile.spec.ts

```diff
@@ -0,0 +1,139 @@
+import { Profile } from '../../../../models';
+
+describe('Personal settings - Profile page', () => {
+  let profileDetails: Profile;
+
+  before(() => {
+    cy.fixture('profile/profile').then((profile) => {
+      profileDetails = profile;
+    });
+  });
+
+  beforeEach(() => {
+    cy.login();
+    cy.get('dib-navbar dib-hamburger-icon').click();
+    cy.get('[routerlink="/profile/account"]').click();
+  });
+
+  it('should edit personal info!', () => {
+    cy.get('dib-profile dib-account div:nth-child(1) > div > div:nth-child(2) > ui-button').click();
```

Comment:
Please write solution without `:nth-child` (please apply same fix on all places)

Reply:
Please do not use the exclamation mark in the test descriptions